docs(gcp): Kubeflow cluster upgrade instructions and revamp of env vars (#2372)
kubeflow/website: content/en/docs/gke/deploy/management-setup.md

    **Note:** Starting from Kubeflow v1.2, we fixed the compatibility problem with Kustomize `v3.2.1+`, so you can now install any Kustomize `v3+`, including the latest Kustomize versions.

    To deploy the latest version of Kustomize on a Linux or Mac machine, run the following commands:

    ```bash
    # Detect your OS and download corresponding latest Kustomize binary
    curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash

    # We need to add the kustomize package to your $PATH env variable
    sudo mv ./kustomize /usr/local/bin/kustomize
    ```

    Then, to verify the installation, run `kustomize version`. You should see `Version:kustomize/vX.Y.Z` in the output if you've successfully deployed Kustomize.

1.  Install [yq](https://github.com/mikefarah/yq#install).

## Setting up a management cluster

This guide assumes the following convention:

- The `${MGMT_DIR}` environment variable contains the path to
  your management directory, which holds your management cluster configuration
  files. For example, `~/kf-deployments/management/`. You can choose any path you would like for the directory `${MGMT_DIR}`.

>   Recommend checking the management directory in source control.

Reviewer: ```suggestion
  You are recommended to check the management directory in source control.
```

- The `${MGMT_NAME}` environment variable contains the name of your management cluster.

However, the environment variables are used purely for command illustration purpose. No tools will assume they actually exists in your terminal environment.

To deploy a management cluster:

1.  Fetch the management blueprint to current directory

    ```bash
    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 "${MGMT_DIR}"
    ```

1.  Change to the Kubeflow directory

    ```bash
    cd "${MGMT_DIR}"
    ```

    Note, all the instructions below assume your current working directory is `${MGMT_DIR}`.

1.  Fetch the upstream management package

    ```bash
    make get-pkg
    ```

1.  Use kpt to set values for the name, project, and location of your management cluster:

    ```bash

>     kpt cfg set -R . name "${MGMT_NAME}"

Reviewer 2: As with other docs in this PR, please replace undefined environment variables that are used once with a placeholder, and then define the variable below the example.

    You can learn more about `kpt cfg set` in [kpt documentation](https://googlecontainertools.github.io/kpt/reference/cfg/set/), or by running `kpt cfg set --help`.

>     Note, you can find out which setters exist in a package and what there current values are by:

Reviewer 2: there -> their

Reviewer 2: by -> by running the following command

    and look into `./build/cluster` folder.

> 1.  Create a kubectl context for the management cluster, it will be called `${MGMT_NAME}`:

Reviewer: ```suggestion
1.  Create a kubectl context for the management cluster called `${MGMT_NAME}`:
```

Reviewer: or "(it will be named `${MGMT_NAME}`):"

    make apply-kcc
    ```

    This step:

>     - Installs Config Connector in your cluster; and

Reviewer 2: ; -> ,

>     This step

Reviewer: ```suggestion
    This step:
```

>     - installs Config Connector in your cluster

Reviewer: ```suggestion
    - Installs Config Connector in your cluster; and
```

>     - creates the Google Cloud service account **${MGMT_NAME}-cnrm-system@${PROJECT}.iam.gserviceaccount.com**

Reviewer: ```suggestion
    - Creates the Google Cloud service account **${MGMT_NAME}-cnrm-system@${PROJECT}.iam.gserviceaccount.com**.
```

> 

Reviewer: for a11y: line 168

-> "and check the `./build/cnrm-install-*` folders." or something like that

> 1. Set the managed project

Reviewer: ```suggestion
1. Set the managed project:
```

   ```bash
   kpt cfg set ./instance managed-project "${MANAGED_PROJECT}"
   ```

> 1. Update the policy

Reviewer: ```suggestion
1. Update the policy:
```

Your management cluster directory contains the following files and directories:

- **Makefile** is a file that defines rules to automate deployment process. You can refer to [GNU make documentation](https://www.gnu.org/software/make/manual/make.html#Introduction) for more introduction. The Makefile we provide is designed to be user maintainable. You are encouraged to read, edit and maintain it to suit your own deployment customization needs.

> - **upstream** is a directory containing kustomize packages for deploying your management cluster

Reviewer 2: Add a period at the end of the sentence. Consider making the sub list a second sentence on this list item.

  - This directory contains the upstream configurations on which your deployment is based on.

- **instance** is a directory that defines user overlays that are applied to the upstream
  configurations in order to customize management cluster for your use case.

  - **cluster** is a kustomize package defining all the Google Cloud resources needed using [gcloud beta anthos apply](https://cloud.google.com/sdk/gcloud/reference/beta/anthos/apply). It can apply some Google Cloud resource types using the same resource definition for Config Connector.

>       - You can edit this kustomize package in order to customize the Google Cloud resources for

Reviewer 2: Consider adding this sentence to the parent list item.
